src/NServiceBus.ContainerTests/When_using_nested_containers.cs

```diff
@@ -26,59 +23,75 @@ public void Instance_per_uow__components_should_be_disposed_when_the_child_conta
         }
 
         [Test]
-        public void Instance_per_uow__components_should_not_be_shared_across_child_containers()
+        public void Instance_per_uow_components_should_yield_different_instances_between_parent_and_child_containers()
         {
             using (var builder = TestContainerBuilder.ConstructBuilder())
             {
-                builder.Configure(typeof(InstancePerUoWComponent),
-                    DependencyLifecycle.InstancePerUnitOfWork);
+                builder.Configure(typeof(InstancePerUoWComponent), DependencyLifecycle.InstancePerUnitOfWork);
 
-                var task1 = Task<object>.Factory.StartNew(() =>
-                {
-                    using (var childContainer = builder.BuildChildContainer())
-                    {
-                        return childContainer.Build(typeof(InstancePerUoWComponent));
-                    }
-                });
-                var task2 = Task<object>.Factory.StartNew(() =>
+                var parentInstance = builder.Build(typeof(InstancePerUoWComponent));
+
+                using (var childContainer = builder.BuildChildContainer())
                 {
-                    using (var childContainer = builder.BuildChildContainer())
-                    {
-                        return childContainer.Build(typeof(InstancePerUoWComponent));
-                    }
-                });
-                Assert.AreNotSame(task1.Result, task2.Result);
+                    var childInstance = childContainer.Build(typeof(InstancePerUoWComponent));
+
+                    Assert.AreNotSame(parentInstance, childInstance);
+                }
             }
         }
 
-
         [Test]
-        public void Should_resolve_child_container_instance_first()
+        public void Instance_per_uow_components_should_yield_different_instances_between_different_instances_of_child_containers()
```

Comment:
this test seems to be the same as the one below (I like this one with the usings better)?
@WilliamBZA Is this one of the tests you added?